Weatherman.py

```diff
@@ -0,0 +1,277 @@
+import argparse
+from csv import DictReader
+from datetime import datetime
+from os import path
+
+White = "\033[37m"
+Red = "\033[31m"
+Blue = "\033[34m"
+
+
+def run_weather_man_month(year, month, operator):
+    file_name = f'Murree_weather_{year}_{month}.txt'
+    read_weather_files(file_name, operator)
+
+
+def read_weather_files(file_name, operator):
+    if path.exists(file_name):
+        with open(file_name, 'r') as file:
+            csv_reader = DictReader(file)
+            maximum_temperatures = []
+            minimum_temperatures = []
+            mean_humidity_values = []
+            for row in csv_reader:
+                maximum_temperatures.append(row['Max TemperatureC'])
+                minimum_temperatures.append(row['Min TemperatureC'])
+                mean_humidity_values.append(row[' Mean Humidity'])
+        calculate_monthly_results(maximum_temperatures, minimum_temperatures,
+                                  mean_humidity_values, operator)
+    else:
+        print(f"{file_name} 'this file does not exists'")
+
+
+def calculate_monthly_results(maximum_temperatures, minimum_temperatures,
+                              mean_humidity_values, operator):
+    if operator == '-a':
+        calculate_average_temperature(maximum_temperatures,
+                                      minimum_temperatures,
+                                      mean_humidity_values)
+    elif operator == '-c':
+        make_bar_chart(maximum_temperatures, minimum_temperatures)
```

Comment:
Instead of checking operators here, we should be moving to right path just at the start of program.